### DISCUSSION TITLE
Issue with Nerd Fonts not Displaying Correctly in LunarVim over SSH

### DISCUSSION THREAD

**Discussion by eyallampel1** [LunarVim/LunarVim#4436] [Q&A]
Hello everyone,

I'm encountering an issue with LunarVim and Nerd Fonts when accessing a remote machine via SSH. I have LunarVim installed on a computer, and I've also installed a Nerd Font (which I've verified is working correctly on the local terminal). However, when I connect to this computer remotely via SSH and launch LunarVim (by typing `lvim` in the terminal), the Nerd Font icons are not displayed correctly. Instead of the proper icons, I see question marks next to options like "find file", "new file", etc., in the LunarVim start menu.

<img width="535" alt="Screenshot 2023-12-08 at 13 54 32" src="https://github.com/LunarVim/LunarVim/assets/17165252/4388fdc6-5644-4f91-a86a-659549a16257">

Here's what I've done so far:
- Installed and verified that Nerd Font is working on the local machine.
- Installed and verified that Nerd Font is working on the remote machine.
- Accessed the remote machine via SSH and tried running LunarVim.
- Encountered the issue where Nerd Font icons are replaced with question marks.

I'm looking for assistance on:
1. Why are the Nerd Font icons not displaying correctly over an SSH connection?
2. Is there a specific configuration or step that I might be missing to ensure Nerd Fonts work correctly in LunarVim over SSH?

Any insights or suggestions would be greatly appreciated. Thank you in advance for your help!

**Comment by LostInTheLogs** [comment]
make sure you installed nerd fonts >= v3, you only need them on the machine where the terminal app is running like any other font
